.github/workflows/TagBot.yml

```diff
@@ -1,7 +1,5 @@
 name: TagBot
 on:
-  schedule:
```

Comment:
@andreasnoack I forgot to remove these lines in #70, seems I copied the updated file incorrectly. `schedule:` is not needed anymore, so removing this trigger will reduce the amount of runs.

All other CI changes were addressed in #70.